### PR TITLE
Show limit

### LINKED ISSUE #91: Showing row limit
To solve #42:

Frontend should show the row limit for every row.

### PR DESCRIPTION
**Pull request by Masclins**
This resolves #91 
Also it changes how the scores are show, so we don't take more space.

### CODE REVIEW

**Comment by kenan-rhoton** [comment]
Doesn't this remove opponent's scores?

**Comment by Masclins** [comment]
No. Now both scores are shown into `#row-scores`, separated by a dash.

File: frontend/src/game/scores.js [hunk 1]
```diff
@@ -7,8 +7,7 @@ module.exports = {
     },
     setScoresByRow(scoresByRowState) {
         scoresByRowState.forEach(function (row, rownum) {
-            document.querySelectorAll("#my-scores .scores-row")[rownum].innerText = row[0];
-            document.querySelectorAll("#opp-scores .scores-row")[rownum].innerText = row[1];
+            document.querySelectorAll("#row-scores .scores-row")[rownum].innerText = row[0] + " - " + row[1];
```
Comment (Masclins) [inline frontend/src/game/scores.js]: Here you can see how this doesn't remove opponent's score

Comment (kenan-rhoton) [reply frontend/src/game/scores.js]: ok, sorry

**Pull request merged**